### PR TITLE
RH7: Add support for RHEL-7.7 distro

### PR DESCRIPTION
**Pull request by johnsongeorge-w**

### CODE REVIEW

File: hv-rhel7.x/hv/Makefile [hunk 1]
```diff
@@ -10,16 +10,20 @@ obj-m	+= hyperv-keyboard.o
...
-
 # Get RHEL/CentOS version information
 rhel_major=$(shell grep -Eoh [0-9]+\.[0-9]+ /etc/{issue,*release} | head -1 | awk -F'.' '{ print $$1 }')
 rhel_minor=$(shell grep -Eoh [0-9]+\.[0-9]+ /etc/{issue,*release} | head -1 | awk -F'.' '{ print $$2 }')
 rhel_release_code=$(shell echo $$(($(rhel_major) << 8 | $(rhel_minor))))
 
+# RDMA should only build on 64 bits arch
+ifeq ($(arch), 64)
+# Network direct driver in LIS is supported for RHEL/CentOS <= 7.6
+build_hv_network_direct=$(shell test $(rhel_release_code) -lt 1799; echo $$?)
```
Comment (santoshx) [inline hv-rhel7.x/hv/Makefile]: Do you need the echo?

Comment (santoshx) [inline hv-rhel7.x/hv/Makefile]: Use tab space for inner lines of condition.

Comment (johnsongeorge-w) [reply hv-rhel7.x/hv/Makefile]: Done.

Comment (johnsongeorge-w) [reply hv-rhel7.x/hv/Makefile]: > Do you need the echo?
Yes

**Pull request merged**